[2024-07-22T00:16:17.172+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-22T00:16:17.213+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task manual__2024-07-22T00:16:15.532041+00:00 [queued]>
[2024-07-22T00:16:17.225+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task manual__2024-07-22T00:16:15.532041+00:00 [queued]>
[2024-07-22T00:16:17.226+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-22T00:16:17.247+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-07-22 00:16:15.532041+00:00
[2024-07-22T00:16:17.255+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=842) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-22T00:16:17.258+0000] {standard_task_runner.py:63} INFO - Started process 843 to run task
[2024-07-22T00:16:17.257+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'first_task', 'manual__2024-07-22T00:16:15.532041+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpu31axozo']
[2024-07-22T00:16:17.259+0000] {standard_task_runner.py:91} INFO - Job 27: Subtask first_task
[2024-07-22T00:16:17.324+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v3.first_task manual__2024-07-22T00:16:15.532041+00:00 [running]> on host eefce25a52e3
[2024-07-22T00:16:17.519+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-07-22T00:16:15.532041+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-22T00:16:15.532041+00:00'
[2024-07-22T00:16:17.520+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-22T00:16:17.579+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-07-22T00:16:17.583+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello World, this is the first task to be done']
[2024-07-22T00:16:17.596+0000] {subprocess.py:86} INFO - Output:
[2024-07-22T00:16:17.598+0000] {subprocess.py:93} INFO - Hello World, this is the first task to be done
[2024-07-22T00:16:17.599+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-07-22T00:16:17.601+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-22T00:16:17.683+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=first_task, run_id=manual__2024-07-22T00:16:15.532041+00:00, execution_date=20240722T001615, start_date=20240722T001617, end_date=20240722T001617
[2024-07-22T00:16:17.717+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-22T00:16:17.753+0000] {taskinstance.py:3498} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-07-22T00:16:17.758+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
